LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/perspective-transforms-equivalence.html

```diff
@@ -9,7 +9,7 @@
 Perspective with different transforms can have small anti-aliasing
 pixel differences, so the test should fuzzy patch to the ref.
 -->
-<meta name="fuzzy" content="maxDifference=0-178;totalPixels=0-538">
+<meta name="fuzzy" content="maxDifference=0-94;totalPixels=0-538">
```

Comment:
This undoes this unexported change: https://github.com/WebKit/WebKit/commit/49ef3002488aa8b83b2f1975816d4172863f1554 from @nikolaszimmermann

Reply:
I will undo it. :-) Plus add any other test expectations in next commit.

Reply:
Yes, unfortunarely there's no auto-export in WebKit, you have to go through the `export-w3c-test-changes` script, this command should do it:

`export-w3c-test-changes -b 290800 -g 49ef3002488aa8b83b2f1975816d4172863f1554 -c`